wire-ios-data-model/Source/Model/Conversation/ZMConversation.swift

```diff
@@ -38,6 +38,10 @@ public extension ZMConversation {
 
     @NSManaged var isPendingInitialFetch: Bool
 
+    /// True if conversation was ran OneOnOneMigrator and migrated to MLS from Proteus Conversation.
+
+    @NSManaged var migratedToMLS: Bool
```

Comment:
question: does this default to `false`? And is it only relevant for mls conversations? Or in other words, it would be invalid to interpret this value for any conversation with the proteus protocol?

Reply:
default is false

Reply:
we'll do a change to mark group conversations that were migrated from proteus